Soccar [Game-Server]/src/nl/soccar/gameserver/SessionController.java

```diff
@@ -130,21 +130,30 @@ private void sendJoinedSessionNotification(Player joinedPlayer, TeamColour joine
         room.getAllPlayers().stream().map(server::getConnectionFromPlayer).forEach(c -> c.send(m));
     }
 
-    public boolean leaveSession(Player player, Session session) {
+    public boolean leaveSession(Player player, TeamColour colour, Session session) {
         Room room = session.getRoom();
-
-        Team teamBlue = room.getTeamBlue();
-        Team teamRed = room.getTeamRed();
-
-        Team team = teamBlue.getPlayers().stream().filter(p -> p.getUsername().equals(player.getUsername())).count() > 0 ? teamBlue : teamRed;
+        
+        Team team = colour == TeamColour.BLUE ? room.getTeamBlue() : room.getTeamRed();
         team.leave(player);
 
         player.setCurrentSession(null);
 
-        PlayerLeftSessionMessage m = new PlayerLeftSessionMessage(player.getUsername(), team.getTeamColour());
+        room.getAllPlayers().stream().map(server::getConnectionFromPlayer).forEach(c -> System.out.println(c));
+        
+        PlayerLeftSessionMessage m = new PlayerLeftSessionMessage(player.getUsername(), colour);
         room.getAllPlayers().stream().map(server::getConnectionFromPlayer).forEach(c -> c.send(m));
+        LOGGER.log(Level.INFO, "Player {0} left the Session.", player.getUsername());
         return true;
     }
+    
+    public void leaveSession(Player player, Session session) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)